.github/workflows/pull-request.yml

```
@@ -240,19 +240,76 @@ jobs:
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks, Test-Application]
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: |
+            image=moby/buildkit:latest
+          cache-from: type=gha
```

actionlint

input "cache-from" is not defined in action "docker/setup-buildx-action@v3". available inputs are "append", "buildkitd-config", "buildkitd-config-inline", "buildkitd-flags", "cache-binary", "cleanup", "config", "config-inline", "driver", "driver-opts", "endpoint", "install", "platforms", "use", "version"

(action)